src/main/java/dev/sapphic/wearablebackpacks/Backpack.java

```diff
@@ -10,131 +10,131 @@
 import net.minecraft.util.collection.DefaultedList;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.BlockView;
-import org.checkerframework.checker.nullness.qual.Nullable;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Please explain why you switched to another static analysis library.

Reply:
I wasn't sure why CheckerFramework was needed.